docs/ui/components/CommandMenu/Items/CommandItemBase.tsx

```diff
@@ -0,0 +1,69 @@
+import { mergeClasses } from '@expo/styleguide';
+import { Command } from 'cmdk';
+import { useState } from 'react';
+
+import { openLink } from '../utils';
+
+import { Tag } from '~/ui/components/Tag';
+
+type Props = {
+  children: React.ReactNode;
+  url: string;
+  onSelect?: () => void;
+  isExternalLink?: boolean;
+  isNested?: boolean;
+  // Props forwarded to Command.Item
+  className?: string;
+  value?: string;
+};
+
+/**
+ * Wrapper for Command.Item that adds copy link on right/ middle click + visual copy indicator.
+ */
+export const CommandItemBase = ({
+  children,
+  url,
+  isExternalLink,
+  isNested,
+  onSelect,
+  className,
+  value,
+}: Props) => {
+  const [copyDone, setCopyDone] = useState(false);
+
+  const copyUrl = () => {
+    navigator.clipboard?.writeText(url);
+    setCopyDone(true);
+    setTimeout(() => setCopyDone(false), 1500);
+  };
+
+  return (
+    <Command.Item
+      className={mergeClasses(className, 'relative')}
```

Comment:
```suggestion
      className={mergeClasses('relative', className)}
```

Styling nit, this order change will make sure that user provided classes will overwrite those already defined inside of the component.